lib/rubocop/cop/rspec/mixin/css_selector.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      # Helps parsing css selector.
+      module CssSelector
+        COMMON_OPTIONS = %w[
+          above below left_of right_of near count minimum maximum between text
+          id class style visible obscured exact exact_text normalize_ws match
+          wait filter_set focused
+        ].freeze
+
+        # @param selector [String]
+        # @return [Boolean]
+        # @example
+        #   id?('#some-id') # => true
+        #   id?('.some-class') # => false
+        def id?(selector)
+          selector.start_with?('#')
+        end
+
+        # @param selector [String]
+        # @return [Boolean]
+        # @example
+        #   attribute?('[attribute]') # => true
+        #   attribute?('attribute') # => false
+        def attribute?(selector)
+          selector.start_with?('[')
+        end
+
+        # @param selector [String]
+        # @return [Array<String>]
+        # @example
+        #   attributes('a[foo-bar_baz]') # => ["foo-bar_baz"]
+        #   attributes('button[foo][bar]') # => ["foo", "bar"]
+        #   attributes('table[foo=bar]') # => ["foo=bar"]
+        def attributes(selector)
```

Comment:
I wonder if it won't make for a better API to return a hash, like
{foo-bar_baz: true, foo: true, baz: true, foo: 'bar'} respectively (from all the examples above). Then you don't really need the _keys and _value methods

Reply:
It looks very good. I fixed it.